src/Microsoft.Azure.SignalR.SignalRService/Scope.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.SignalR.SignalRService
+{
+    public enum Scope
+    {
+        Client, // scope for  client to connect to a hub
```

Comment:
>Client [](start = 8, length = 6)

add detailed comments for the user `///` style ones

Reply:
Ok